packages/vaex-core/vaex/dataset.py

```diff
@@ -5754,3 +5754,13 @@ def add_column(self, name, data):
         super(DatasetArrays, self).add_column(name, data)
         self._length_unfiltered = int(round(self._length_original * self._active_fraction))
         # self.set_active_fraction(self._active_fraction)
+
+    @property
+    def values(self):
+        """
+        Return a numpy representation of the Dataset.
+        Only the values in the Dataset are returned, the column labels are removed.
+
+        Does not preserve the mask if any of the converted columns are masked arrays.
```

Comment:
This docstring should be the same as `__array__`, or refer to it.  Also, always start directly after """ with see https://www.python.org/dev/peps/pep-0257/. Now you start on the second, so it will not show up with a simple ds.values?
In general I think sth like this should be good:
```
"""Gives a full memory copy of the dataset into a 2d numpy array of shape (#rows, #columns).
Note that the memory order is fortran, so all values of 1 column are contiguous in memory, for performance reasons

Note this returns the same result as:

>>> np.array(ds)

Except this can optionally take a dtype argument

Masked values are replaced by NaN's when the dtype if float, for other types the behaviour is undefined.
"""

And update the `__array__` docstring accordingly